api/tests/src/api/users/test_user_update_saved_search.py

```diff
@@ -0,0 +1,74 @@
+import uuid
+
+import pytest
+
+from src.db.models.user_models import UserSavedSearch, UserTokenSession
+from tests.src.db.models.factories import UserSavedSearchFactory
+
+
+@pytest.fixture
+def saved_search(enable_factory_create, user, db_session):
+    search = UserSavedSearchFactory.create(
+        user=user, name="Save Search", search_query={"keywords": "python"}
+    )
+    db_session.commit()
+    return search
```

Comment:
The [DB session commits automatically for each factory](https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/src/db/models/factories.py#L317), don't need to commit.

```suggestion
@pytest.fixture
def saved_search(enable_factory_create, user, db_session):
    return UserSavedSearchFactory.create(
        user=user, name="Save Search", search_query={"keywords": "python"}
    )
```

Reply:
Done